dependency_support/com_github_facebook_zstd/bundled.BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 The XLS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+# Builds everything together similarly to the zstd cmake file:
+# https://github.com/facebook/zstd/blob/dev/build/cmake/lib/CMakeLists.txt
+# but with legacy support and defines as in the zstd BUCK file:
+# https://github.com/facebook/zstd/blob/dev/lib/BUCK
+
+cc_library(
+    name = "zstd",
+    srcs = glob([
+        "lib/common/*.h",
+        "lib/common/*.c",
+        "lib/compress/*.h",
+        "lib/compress/*.c",
+        "lib/decompress/*.h",
+        "lib/decompress/*.c",
+        "lib/decompress/*.S",
+        "lib/deprecated/*.h",
+        "lib/deprecated/*.c",
+        "lib/dictBuilder/*.h",
+        "lib/dictBuilder/*.c",
+        "lib/legacy/*.h",
+        "lib/legacy/*.c",
+    ]),
+    hdrs = [
+        "lib/zstd.h",
+    ],
+    strip_include_prefix = "lib",
+    linkopts = [
+        "-pthread",
```

Comment:
Is this necessary? I'd have thought not

Reply:
It looks like disabling multithread builds does not affect our tests which use zstd library. I've updated the build rule to build library without multithread support.